### PR TITLE
fix: scroll bar missing and some ux issue

### LINKED ISSUE #2532: can not use a list entity without synonyms
## Describe the bug
When I use list entity with synonyms, the bot works OK.
image

But when I use a list entity without synonyms, the bot failed to start. I think a list entity without synonyms should be valid?
image
image

## Browser

- [x] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

### LINKED ISSUE #2595: Used checkmark on Bot Responses table is not interact-able and therefore should not be a button
## Describe the bug
On the Bot Responses page, the 'Used' checkmark is a button, however; there is no interactable behavior, so it should not be a button, but an icon with tabindex/aria-label

## Version
latest master

## Browser

- [x] Chrome
- [ ] Safari
- [ ] Firefox
- [x] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

## To Reproduce

1. Open Composer
2. Open a ToDo bot
3. Click Bot Responses button
4. Note that the 'Used' checkmark is a button. For example it has a hover and click state. You can also check the element.

## Expected behavior

When interacting with Bot Responses, the 'Used' checkmark should be an icon, not an interact-able button.
## Screenshots

image

### LINKED ISSUE #2603: Scroll bar is missing in lu/lg all up
image

### LINKED ISSUE #2633: Need to align the 'BeenUsed' mark for LU and LG
Synced with @zhixzhan , BeenUsed will be "checked" if such template has been used in dialog file for LG. eg. send-activity with #bfd-xxx. But for LU text input hoisting, no "checked" if this has been used .
image
image

### PR DESCRIPTION
**Pull request by zhixzhan**
## Description
1. fix scroll bar missing in all up view left navigate bar.
2. fix beenUsed icon should not be interact-able.
3. for LU all up view  beenUsed  calculation are no longer explicitly correct, hide this column for now.
4. fix Luis publish error

## Task Item

closes #2603 
closes #2633
closes #2595 
closes #2532 

## Screenshots

Screen Shot 2020-04-14 at 4 59 06 PM

### CODE REVIEW

File: Composer/packages/client/src/pages/language-understanding/table-view.tsx [hunk 1]
```diff
@@ -148,19 +148,23 @@ const TableView: React.FC<TableViewProps> = props => {
...
+      //   isCollapsable: true,
+      //   data: 'string',
+      //   onRender: item => {
+      //     return item.used ? (
+      //       <FontIcon iconName="Accept" aria-label={formatMessage('Used')} className={iconClass} />
+      //     ) : (
+      //       <div />
+      //     );
+      //   },
+      // },
```
Comment (corinagum) [inline Composer/packages/client/src/pages/language-understanding/table-view.tsx:167]: Delete uncommented code

Comment (a-b-r-o-w-n) [reply Composer/packages/client/src/pages/language-understanding/table-view.tsx:167]: I think the intention is to bring this back after the bug is fixed.

**Comment by zhixzhan** [comment]
> Before:
> image
> 
> After:
> image
> 
> Checkmarks and vertical 'more' button are no longer vertically aligned.

updated, 
image

File: Composer/packages/client/src/pages/language-understanding/table-view.tsx (hunk 1 above)
Comment (zhixzhan) [reply Composer/packages/client/src/pages/language-understanding/table-view.tsx:167]: Yes, when we can explicitly calculate a lu intent is used or not.

**Pull request merged**